transactionparser/src/channeldetails.js

```diff
@@ -0,0 +1,38 @@
+const normalizeNodes = require('./utils').normalizeNodes;
+const dropsToXRP = require('./utils').dropsToXRP;
+const BigNumber = require('bignumber.js');
+
+function summarizePaymentChannel(node) {
+
+      const final = (node.diffType === 'CreatedNode') ? node.newFields : node.finalFields;
+      const prev = node.previousFields || {};
+
+      return {
+        channelId: node.ledgerIndex,
```

Comment:
@adrianhopebailie Why is this called `channelId` rather than `channel_id`?

Reply:
Why is any of it snake_case ? Odd :)